editor/components/inserter/menu.js

```diff
@@ -1,46 +1,100 @@
 /**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import uuid from 'uuid/v4';
```

Comment:
I was thinking it might be helpful to move this logic into the block API, like:

```
wp.blocks.createBlock( 'core/text', {
	content: 'Hello World'
} );
```

What do you think?